src-tauri/src/algorithms/stoer_wagner.rs

```diff
@@ -93,6 +118,15 @@ mod tests {
 
         let mut correct_mincut_weight_count = 0;
 
+        let graph_sw = &mut StoerWagnerGraph::new(g.clone());
+        let _mincut = stoer_wagner(graph_sw);
+
+        let nodes = vec![u, v, w, x, y, z, a, b];
+        let (s, t) = recover_mincut(graph_sw, nodes);
+
+        println!("s: {:?}", s);
```

Comment:
Might want to put this behind a non-production (debug) flag, we don't want to be logging in prod